step_open_log_writer.go

```diff
@@ -49,8 +49,9 @@ func (s *stepOpenLogWriter) writeUsingWorker(state multistep.StateBag, w io.Writ
 		_, _ = writeFold(w, "worker_info", []byte(strings.Join([]string{
 			"\033[33;1mWorker information\033[0m",
 			fmt.Sprintf("hostname: %s", hostname),
-			fmt.Sprintf("version: %s", VersionString),
+			fmt.Sprintf("version: %s https://github.com/travis-ci/worker/tree/%s", VersionString, RevisionString),
```

Comment:
Not really. It felt a little strange to put the URL in the code like that, but I think it's the right way to do it.